src/%IPM/Utils/Module.cls

```diff
@@ -1149,11 +1149,11 @@ ClassMethod LoadNewModule(pDirectory As %String, ByRef pParams, pRepository As %
 		If $Get(pParams("DeveloperMode"),0) {
 			Set tCurrentSourceControl = ##class(%Studio.SourceControl.Interface).SourceControlClassGet()
 			If (tCurrentSourceControl = "") {
-				Set tCompositeSCClass = "%IPM.PackageManager.Developer.Extension.Composite"
+				Set tCompositeSCClass = "%IPM.ExtensionBase.Composite"
 				$$$ThrowOnError(##class(%Studio.SourceControl.Interface).SourceControlClassSet(tCompositeSCClass))
-				Set tPrimaryExtension = "%IPM.PackageManager.Developer.Extension.SourceControl.ISC"
-				Set tSecondaryExtensions = $ListBuild("%IPM.PackageManager.Developer.Extension.PackageManager")
-				$$$ThrowOnError(##class(%IPM.PackageManager.Developer.Extension.Composite).SetExtensionClasses(tPrimaryExtension,tSecondaryExtensions))
+				Set tPrimaryExtension = "%IPM.Extension.SourceControl.Perforce.Main"
```

Comment:
Should not reference these classes here as they are in zpm-perforce and zpm-studio-ui repos

Reply:
Created https://usjira.iscinternal.com/browse/HSIEO-6334 to handle this